components/Composition/FAnnouncementCard.jsx

```diff
@@ -11,86 +10,116 @@ import icons from 'themes/icons';
+  return (
+    <TouchableWithoutFeedback onPress={() => navigation.push(link)}>
+      <View style={{
+        padding: sizes.PADDING_5,
+      }}
       >
-        <View style={{
-          minHeight: height,
-          maxHeight: height,
-        }}
+        <FCard
+          width={width}
+          paddingHorizontal={sizes.PADDING_10}
+          paddingVertical={sizes.PADDING_10}
         >
-          <FImage
-            imagePath={images.RABBIT_BLACK()}
-            imageStyle={styles.image}
-            height={sizes.HEIGHT_150}
-            resizeMode="cover"
-            width="100%"
-          />
-          <FHeading
-            title={data.title}
-            weight={fonts.HEADING_WEIGHT_BOLD}
-            size={fonts.HEADING_MEDIUM}
-            marginBottom={sizes.MARGIN_5}
-            style={{ marginTop: sizes.MARGIN_8 }}
-            ellipsizeMode="tail"
-            numberOfLines={1}
-          />
-          <FHeading
-            title={data.description}
-            marginBottom={sizes.MARGIN_10}
-            color={colors.DARK_GRAY}
-            ellipsizeMode="tail"
-            numberOfLines={2}
-            size={fonts.HEADING_NORMAL}
-            weight={fonts.HEADING_WEIGHT_REGULAR}
-          />
-          <FHeadingWithIcon
-            icon={icons.LOCATION_OUTLINE}
-            iconColor={colors.PRIMARY}
-            title={data.locationName}
-            titleStyle={styles.text}
-            titleColor={colors.DARK_GRAY}
-            numberOfLines={2}
-            iconPlacement={placements.LEFT}
-            iconSize={sizes.ICON_20}
-            titleSize={fonts.HEADING_SMALL}
-            titleWeight={fonts.HEADING_WEIGHT_REGULAR}
-          />
-          <FHeadingWithIcon
-            icon={icons.CALENDAR}
-            iconColor={colors.PRIMARY}
-            title={data.date}
-            titleStyle={styles.text}
-            titleColor={colors.DARK_GRAY}
-            size={fonts.HEADING_MEDIUM}
-            weight={fonts.HEADING_NORMAL}
-            iconPlacement={placements.LEFT}
-            iconSize={sizes.ICON_20}
-            titleSize={fonts.HEADING_SMALL}
-            titleWeight={fonts.HEADING_WEIGHT_REGULAR}
-          />
-        </View>
-      </FCard>
-    </View>
-  </TouchableWithoutFeedback>
-);
+          <View style={{
+            minHeight: height,
+            maxHeight: height,
+          }}
+          >
+            <FImage
+              imagePath=""
+              networkImageUrl={data.photos[0]}
+              imageStyle={styles.image}
+              height={sizes.HEIGHT_150}
+              resizeMode={sizes.COVER}
+              width={sizes.WIDTH_FULL}
+            />
+            <FHeading
+              title={data.title}
+              weight={fonts.HEADING_WEIGHT_BOLD}
+              size={fonts.HEADING_MEDIUM}
+              marginBottom={sizes.MARGIN_5}
+              style={{
+                marginTop: sizes.MARGIN_8,
+                ...titleStyle,
```

Comment:
po co tutaj to? Wszystkie style powinny być tutaj na sztywno zrobione, dlatego że każda ta karta ma wyglądać tak samo. Nie wydaje mi się że bedziemy dodawać jakieś style iterując po liście :D